runtime/controller.go

```diff
@@ -1360,24 +1358,35 @@ func (c *Controller) processCompletedInvocation(inv *invocation) error {
 		c.Logger.Info("Reconciled resource", logArgs...)
 	}
 
-	// Emit event unless it was a cancellation.
-	if inv.cancelledOn.IsZero() {
-		eventArgs := []attribute.KeyValue{
-			attribute.String("name", inv.name.Name),
-			attribute.String("type", PrettifyResourceKind(inv.name.Kind)),
-			attribute.Int64("elapsed_ms", elapsed.Milliseconds()),
-		}
-		if inv.isDelete {
-			eventArgs = append(eventArgs, attribute.Bool("deleted", true))
-		}
-		if inv.isRename {
-			eventArgs = append(eventArgs, attribute.Bool("renamed", true))
-		}
-		if inv.result.Err != nil {
-			eventArgs = append(eventArgs, attribute.String("error", inv.result.Err.Error()))
+	commonDims := []attribute.KeyValue{
+		attribute.String("resource_id", inv.name.Name),
+		attribute.String("resource_type", PrettifyResourceKind(inv.name.Kind)),
+	}
+	if inv.isDelete {
+		commonDims = append(commonDims, attribute.Bool("is_deleted", true))
+	}
+	if inv.isRename {
+		commonDims = append(commonDims, attribute.Bool("is_renamed", true))
+	}
+
+	if !inv.cancelledOn.IsZero() {
+		commonDims = append(commonDims, attribute.String("reconcile_status", "Cancelled"))
+	} else if inv.result.Err != nil {
+		if errors.Is(inv.result.Err, context.Canceled) {
+			commonDims = append(commonDims, attribute.String("reconcile_status", "Cancelled"))
```

Comment:
Given the key is snake case, consider also using snake case for the values (this seems like pascal case or title case). E.g. `canceled` instead of `Cancelled`.